ci/cpe.yml

```diff
@@ -0,0 +1,27 @@
+include:
+  - remote: 'https://gitlab.com/cscs-ci/recipes/-/raw/master/templates/v2/.ci-ext.yml'
+
+stages:
+  - run
+
+run rfm_test:
+  extends: .container-runner-hohgant
+  stage: run
+  tags: ['hohgant-login-baremetal']
+  variables:
+    GIT_STRATEGY: fetch
+  script:
+    - rm -fr venv_reframe_cpe
+    - python3 -m venv venv_reframe_cpe
+    - source venv_reframe_cpe/bin/activate
+    - pip install reframe-hpc
+    - export RFM_AUTODETECT_XTHOSTNAME=1
+    - export RFM_REMOTE_DETECT=0
```

Comment:
Why are do we skip the autodetection of the remote partitions? Isn't it supposed to be used in our generic tests?